README.md

```diff
@@ -54,7 +57,9 @@ Using Ruby...
 Write your code here...
 
 ```ruby
-# code here
+town[:residents].slice!(1)
+town[:castle][:guest] << "Belle"
+return guests:
 ```
```

Comment:
don't need a return since you aren't in a method but otherwise.  `:guest` should be `:guests` but your syntax is correct. 

you could also do `town[:castle][:guest] << town[:residents].slice!(1)` since slice returns the element(s) taken out of or copied from the array!